### ISSUE TITLE
Transitioning to a map with an opening message will always display the message even if the message has been seen

### ISSUE THREAD

**Issue by Alphanumber** [ASteinheiser/react-rpg.com#42]
The player can transition back and forth between maps and the opening message will display every time when transitioning to the map with a message.

**Comment by ASteinheiser** [comment]
Good catch! Maybe we add a flag to each map called "messageOpened" or something, if a message was shown.
